Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -63,7 +63,8 @@ @interface WMFExploreViewController () <WMFExploreSectionSchemaDelegate,
     UIViewControllerPreviewingDelegate,
     WMFAnalyticsContextProviding,
     WMFAnalyticsViewNameProviding,
-    WMFColumnarCollectionViewLayoutDelegate>
+    WMFColumnarCollectionViewLayoutDelegate,
+    UINavigationControllerDelegate>
```

Comment:
@joewalsh I don't see any delegate methods implemented though - are we missing some behavior?